client/client.go

```diff
@@ -195,6 +196,42 @@ func CanCreateSSHConnection(address, username, password string, config *Config)
 	return true, cli, nil
 }
 
+func CheckSSHBanner(address string, cfg *Config) (bool, int, error) {
+	var port string
+	if strings.Contains(address, ":") {
+		addressAndPort := strings.Split(address, ":")
+		if len(addressAndPort) != 2 {
+			return false, 1, errors.New("invalid address for ssh, format must be host:port")
```

Comment:
```suggestion
			return false, 1, errors.New("invalid address for ssh, format must be host:port")
```
I believe the format should be `ssh://host:port`, not just `host:port`, but please double check when you have a moment

Reply:
Yeah, originally I did that because it was already trimmed, but I'll change it.